corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -1208,6 +1214,7 @@ hqDefine('app_manager/js/detail-screen-config.js', function () {
                                 spec.sortRows[j].field,
                                 spec.sortRows[j].type,
                                 spec.sortRows[j].direction,
+                                spec.sortRows[j].header[this.lang],
```

Comment:
Using header as the dict and referring the _value_ part in _key: value_ pair as display.
Doing this to have the same representation as DetailColumn:
1. Reuse the implementation of detail_column_header_locale method.
2. There seems efficient way to differentiate only sort column [at](https://github.com/dimagi/commcare-hq/pull/12523/files#diff-ae61842a4e76ffee74843e118b718a68R103) since we have removed the SortOnlyDetailColumn class.
Changes needed :
```
def locale_id(self):
        if self.column.format == 'invisible':
            return self.id_strings.invisible_column_header_locale(
                self.module, self.detail_type, self.column,
            )
        else:
            return self.id_strings.detail_column_header_locale(
                self.module, self.detail_type, self.column,
            )
```
and add
```
@pattern('m%d.%s.%s_%s_%d.display')
def invisible_column_header_locale(module, detail_type, column):
    field = column.field.replace('#', '')
    return u"m{module.id}.{detail_type}.{d.model}_{field}_{d_id}.display".format(
        detail_type=detail_type,
        module=module,
        d=column,
        field=field,
        d_id=column.id + 1
    )
```

Not sure if it would be worth the change? And is the check for column format reliable?